package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "cho-cho",
+  "version": "1.0.0",
+  "description": "## Instructions",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha"
```

Comment:
Almost. While this works here when you change the file name, it won't work for most projects which won't have a single file we can rename. Instead we should be providing the pattern that mocha would use to find our tests as we saw in slide 52.

`"test": "mocha 'tests.js'"`